HACKING.md

```diff
@@ -190,6 +195,14 @@ specify its name in an argument:
 
     $ ./admin/RunReports.pl DuplicateArtists
 
+### Adding a new report
+
+1. Create new nodule in */lib/MusicBrainz/Server/Report/* directory.
```

Comment:
s/nodule/module/

Also, either "in the foo/ directory" or "in foo/" would be slightly more grammatical (just not "in foo/ directory").